static/stable/stage/services/services.json

```diff
@@ -51,7 +51,8 @@
             "icon": "AnsibleIcon",
             "description": "Register your Ansible Automation Platform systems with the Red Hat Insights Client to view them on the Red Hat Hybrid Cloud Console."
           },
-          "ansible.tasks"
+          "ansible.tasks",
+          "ansible.ansibleWisdomAdminDashboard"
```

Comment:
Likewise; navigation for `stable/stage` is defined [here](https://github.com/RedHatInsights/chrome-service-backend/blob/e8245ee64406f0f055209c00215a61d82df668c2/static/stable/stage/navigation/ansible-navigation.json#L129).